### PR TITLE
refactor(app): border radius full overrides

### PR DESCRIPTION
**Pull request by mjhuff**
Closes [EXEC-334](https://opentrons.atlassian.net/browse/EXEC-334)

# Overview

As with the other PRs, see the ticket for the figma link. This handles all the one-off cases for border-radius full. 

# Test Plan
- Mel will test, but feel free to look at Figma! This does not include the DropDownMenu, since @ncdiehl11 will be tackling that as a part of the larger component unification work.

# Risk assessment
low

[EXEC-334]: https://opentrons.atlassian.net/browse/EXEC-334?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/atoms/MenuList/DropdownMenu.tsx [hunk 1]
```diff
@@ -17,6 +17,11 @@ import {
 import { StyledText } from '../text'
 import { MenuItem } from './MenuItem'
 
+const SELECTED_OPTION_TEXT_STYLE = css`
+  ${TYPOGRAPHY.pSemiBold};
+  margin-left: ${SPACING.spacing4};
+`
```
Comment (koji) [inline app/src/atoms/MenuList/DropdownMenu.tsx]: @nickcrider

**Review by koji — COMMENTED** [review]
Nick is updating `DropdownMenu` for the unification.
Probably getting his insight would be great.

**Comment by mjhuff** [comment]
> Nick is updating `DropdownMenu` for the unification. Probably getting his insight would be great.

Thanks @koji, good call! After speaking with @ncdiehl11, he'll take the DropDownMenu. I've removed that commit from my PR.

**Review by koji — APPROVED** [review]
lgtm

**Pull request merged**